src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -153,6 +154,32 @@ public static void Visit(this MarkdownObject? obj, MarkdownVisitContext context,
                     context.FileStack.Pop();
                     break;
 
+                case TripleColonBlock tripleColonBlock:
+                    string? target = null;
+                    if (tripleColonBlock.GetAttributes().Properties.Any(p => p.Key == "data-pivot"))
+                    {
+                        target = "pivot";
+                    }
+                    else
+                    {
+                        target = tripleColonBlock.GetAttributes().Properties.FirstOrDefault(p => p.Key == "data-target").Value;
+                    }
```

Comment:
target should have higher priority than pivot for our validation case?

Reply:
ok